### PR TITLE
feat: add github action to sync open api docs

### LINKED ISSUE #117: Update https://docs.openpayments.guide via GitHub action
https://blog.readme.com/lights-camera-github-actions/

### PR DESCRIPTION
**Pull request by sabineschaller**
fixes #117 

Sync open api spec files via github actions

### CODE REVIEW

File: .github/workflows/docs2readme.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Deploy docs to readme
+
+on:
+  push:
+    branches: -master
```
Comment (wilsonianb) [inline .github/workflows/docs2readme.yaml:5]:
```suggestion
    branches:
      - master
```

Comment (wilsonianb) [inline .github/workflows/docs2readme.yaml:5]:
```suggestion
    branches:
      - master
```

**Review by wilsonianb — APPROVED** [review]

File: .github/workflows/docs2readme.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Deploy docs to readme
+
+on:
+  push:
+    branches:
+      - master
+
+obs:
```
Comment (wilsonianb) [inline .github/workflows/docs2readme.yaml:8]:
```
jobs:
```
https://github.com/interledger/open-payments/actions/runs/2344378043

Comment (sabineschaller) [reply .github/workflows/docs2readme.yaml:8]: Fixed in [868df19](https://github.com/interledger/open-payments/commit/868df198a3ecc501d73aa7ac6468c9e4afb20e9d). Sorry for being so sloppy.

**Pull request merged**